functions/src/definitions/common/parameters/thresholds.json

```diff
@@ -9,8 +9,9 @@
   "isIrrelevant": 0.5,
   "isUnsure": 0.5,
   "isSus": 0.5,
-  "falseUpperBound": 1.5,
-  "misleadingUpperBound": 3.5,
+  "isSatire": 0.5,
```

Comment:
Should these threshold values be updated as well? It seems like the original threshold values are within the bounds of the original score 0-5. So checking if they should similarly be within the new bounds (e.g. 1.2 or 1.5)

Reply:
Oh no! These is*** thresholds determine the proportion of votes that must be on a given category for the message to be considered that category. So they aren't related to the rescaling.

The only two that need to be adjusted are falseUpperBound and misleadingUpperBound which have been changed already.